azure-defender-easm/add-easm-assets-to-tracker/cloudbuild.yaml

```diff
@@ -0,0 +1,34 @@
+steps:
+  - name: "gcr.io/cloud-builders/docker"
+    id: generate-image-name
+    entrypoint: "bash"
+    dir: azure-defender-easm/add-easm-assets-to-tracker
+    args:
+      - "-c"
+      - |
+        echo "gcr.io/$PROJECT_ID/azure-defender-easm/add-easm-assets-to-tracker:$BRANCH_NAME-$SHORT_SHA-$(date +%s)" > /workspace/imagename
```

Comment:
We've migrated away from the `gcr.io` repo to `northamerica-northeast1-docker.pkg.dev/track-compliance/tracker`